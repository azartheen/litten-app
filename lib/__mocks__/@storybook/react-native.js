// __mocks__/@storybook/react-native.js

export const getStorybookUI = jest.fn()

export const configure = jest.fn()
