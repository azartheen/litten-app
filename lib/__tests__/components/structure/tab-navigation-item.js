/**
 * @format
 */

import { View, Text } from 'react-native'
import TabNavigationItem from 'structure/tab-navigation-item'
import TestRenderer from 'react-test-renderer'

const CustomComponent = (props) => <View />

describe('Snapshot test for the "TabNavigationItem" component', () => {
  it('renders correctly', () => {
    const props = {
      key: 'SCREEN_TAB_NAV_PROFILE',
      name: 'SCREEN_TAB_NAV_PROFILE',
      component: (
        <View>
          <Text>Hello there</Text>
        </View>
      ),
      options: {
        tabBarIcon: CustomComponent,
        tabBarAccessibilityLabel: 'accessibility.tabBar.profile',
      },
    }
    const element = TestRenderer.create(
      <TabNavigationItem {...props} />,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
