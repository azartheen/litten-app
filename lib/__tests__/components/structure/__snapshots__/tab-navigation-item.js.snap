// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "TabNavigationItem" component renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "paddingBottom": 0,
      },
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 3,
          "height": 6,
          "marginTop": 8,
          "width": 6,
        },
        undefined,
      ]
    }
  />
</View>
`;
